### PR TITLE
96 add flood watches and warnings layer

### PR DESCRIPTION
**Pull request by lilyhoutman**
same as the other: should we do something about the legend?

### CODE REVIEW

File: src/components/Map.vue [hunk 1]
```diff
@@ -756,12 +762,12 @@ export default {
         } else {
           if (e.layer.getPopup() != undefined) {
             e.layer.getPopup().setContent(this.aqPopupContent, {
-              maxWidth: 600,
+              mixWidth: 600,
```
Comment (lprivette) [inline src/components/Map.vue:765]: should be minWidth?

```suggestion
              minWidth: 600,
```

File: src/components/Map.vue [hunk 2]
```diff
@@ -1196,16 +1216,22 @@ export default {
         }
       }
     },
+    getFwwLayer() {
+      this.fwwLayer = esri.dynamicMapLayer({
+        url: "https://idpgis.ncep.noaa.gov/arcgis/rest/services/NWS_Forecasts_Guidance_Warnings/watch_warn_adv/MapServer",
+        layers: [0, 1],
+        format: "image/png",
```
Comment (lprivette) [inline src/components/Map.vue:1223]:
```suggestion
        f: "image/png",
```
looking at their docs https://esri.github.io/esri-leaflet/api-reference/layers/dynamic-map-layer.html I think we want 'format' to be 'f', should help with layer not loading

File: src/components/Map.vue [hunk 3]
```diff
@@ -1196,16 +1216,22 @@ export default {
...
+      });
+      this.fwwLayer.addTo(this.map);
+    },
     getRadarLayer() {
       this.radarLayer = esri.dynamicMapLayer({
         url: "https://idpgis.ncep.noaa.gov/arcgis/rest/services/NWS_Observations/radar_base_reflectivity/MapServer",
         // 0: NFHL Availability, 3: FIRM Panels, 14: Cross Sections, 27: Flood Hazard Boundaries, 28: Flood Hazard Zones
         layers: [3],
         format: "image/png",
```
Comment (lprivette) [inline src/components/Map.vue:1232]:
```suggestion
        f: "image/png",
```

same here too

Comment (lilyhoutman) [reply src/components/Map.vue]: this is exactly what needed to happen and u are a genius

**Pull request merged**